pytket/tests/zx_diagram_test.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from math import isclose, pow
+from math import isclose, pow  # noqa: A004
```

Comment:
I suggest just removing the `pow` import and using the built-in one; as used in this file there is no difference.

Reply:
done in the merge commit